src/pages/BookPage/bookPageWithoutLogin.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import './style.scss';
+import bookCover from './bookCover.jpg';
+import { MDBContainer, MDBRow, MDBCol, MDBBtn, MDBIcon } from 'mdbreact';
+import { useTranslation } from 'react-i18next';
+const BookPage = () => {
+  const { t } = useTranslation();
+  const bookName = 'Mind Without Body';
+  const authortName = 'Ahmed khaled tofeq';
+  const bookType = 'Fantasy';
+  const publishDate = '2017';
+  const isLoggedIn = false;
+  return (
+    <MDBContainer className="mt-5">
+      <MDBRow>
+        <MDBCol sm="6" size="12">
+          <div className="mt-5">
+            <h1 className="blue-text"> {bookName}</h1>
+            <span className="gray-text">{authortName}</span>
+            <br />
+            <div className="mt-2 yellow-star">
+              <MDBIcon icon="star"></MDBIcon>
+              <MDBIcon icon="star"></MDBIcon>
+              <MDBIcon icon="star"></MDBIcon>
+              <MDBIcon icon="star"></MDBIcon>
+              <MDBIcon icon="star"></MDBIcon>
+            </div>
```

Comment:
you shouldn't hard code this part and you should receive the rating as data from somewhere and using loop to render the stars or you could make them a separate component and follow the same logic there

Reply:
okay i will ask zeena first if she did it if she not i will do it